src/constants.ts

```diff
@@ -0,0 +1,6 @@
+// Capabilities
+export enum Capabilities {
+  AtomicBatch = 'atomicBatch',
+  AuxiliaryFunds = 'auxiliaryFunds',
+  PaymasterService = 'paymasterService',
+}
```

Comment:
we'll still need to use this enum for any capabilities we support in OnchainKit but now developers can use non-OCK supported capabilities

```
const { walletCapabilities } = useOnchainKit()

if (walletCapabilities[someOtherCapability]){
  ...
}
```

Reply:
For each of them, we should probably add a comment that links on where the EIP is.

Just because I can imagine a convo in 2 months where someone else tries to figure out what those are and they take a few to find the correct docs.